packages/ketcher-react/src/script/ui/component/cliparea/cliparea.jsx

```diff
@@ -73,6 +74,23 @@ class ClipArea extends Component {
           event.preventDefault();
         }
       },
+      keydown: async (event) => {
+        if (this.props.focused() && this.props.onPaste) {
```

Comment:
I propose to remove nesting. The simplest way:
```
if (!this.props.focused() || !this.props.onPaste)) {
 return
}

if (...)
```

Reply:
done